src/update_metadata.rs

```diff
@@ -27,6 +27,88 @@ use crate::decode::{decode, get_metadata_pda};
 use crate::limiter::create_rate_limiter;
 use crate::parse::{convert_local_to_remote_data, parse_keypair};
 
+pub fn update_name_one(
+    client: &RpcClient,
+    keypair: &String,
+    mint_account: &String,
+    new_name: &String,
+) -> Result<()> {
+    let parsed_keypair = parse_keypair(keypair)?;
+    let data_with_old_name = decode(client, mint_account)?.data;
+    let new_data: Data = Data {
+        creators: data_with_old_name.creators,
+        seller_fee_basis_points: data_with_old_name.seller_fee_basis_points,
+        name: new_name.to_owned(),
+        symbol: data_with_old_name.symbol,
+        uri: data_with_old_name.uri,
+    };
+
+    update_data(client, &parsed_keypair, mint_account, new_data)?;
+    Ok(())
+}
+
+pub fn update_symbol_one(
+    client: &RpcClient,
+    keypair: &String,
+    mint_account: &String,
+    new_symbol: &String,
+) -> Result<()> {
+    let parsed_keypair = parse_keypair(keypair)?;
+    let data_with_old_symbol = decode(client, mint_account)?.data;
+    let new_data: Data = Data {
+        creators: data_with_old_symbol.creators,
+        seller_fee_basis_points: data_with_old_symbol.seller_fee_basis_points,
+        name: data_with_old_symbol.name,
+        symbol: new_symbol.to_owned(),
+        uri: data_with_old_symbol.uri,
+    };
+
+    update_data(client, &parsed_keypair, mint_account, new_data)?;
+    Ok(())
+}
+
+pub fn update_creator_by_position(
+    client: &RpcClient,
+    keypair: &String,
+    mint_account: &String,
+    new_creator: &String,
+    new_share: &u8,
+    position: &u64,
+) -> Result<()> {
+    if new_share > &100 && new_share < &0 {
+        error!("Invalid creator share")
+    }
+
+    let parsed_keypair = parse_keypair(keypair)?;
+    let data_with_old_creators = decode(client, mint_account)?.data;
+    let new_creator_pb = Pubkey::from_str(&new_creator)?;
+    let is_verified = parsed_keypair.pubkey().eq(&new_creator_pb);
+    let new_creator = Creator {
+        address: new_creator_pb,
+        share: *new_share,
+        verified: is_verified,
+    };
+
+    let new_creators = match data_with_old_creators.creators {
+        Some(mut old_creators) => {
+            let _creator = std::mem::replace(&mut old_creators[*position as usize], new_creator);
```

Comment:
`std::mem::replace` is a clever solution for swapping out creators that already exist. I think we want the API to also allow adding creators to positions that don't exist yet though. If there's a creator's array with only one creator (at position 0) I think it makes sense to allow someone to add an additional creator at position 1. What do you think?